net-mail/email/email-3.1.3-r2.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Advanced CLI tool for sending email"
+HOMEPAGE="https://github.com/deanproxy/eMail"
+SRC_URI="http://www.cleancode.org/downloads/${PN}/${P}.tar.bz2"
+LICENSE="GPL-2"
+SLOT="0"
+
+KEYWORDS="~alpha amd64 x86"
```

Comment:
Yes I actually wanted to ask you this but forgot. This is so minor I just wanted it to put it in without revbump. But I can do a revbump at least.

Reply:
We need to revbump for these just in case it led to misconfiguration ("configure" silently got the wrong result), but it's ok to just git mv when it's trivial, so people rebuild, but no need to do the whole 30-days-in-~arch thing.

Reply:
so always revbump, but for this trivial stuff always no ~arch and delete the old ebuild immideatly? got it .